internal/config/config.go

```diff
@@ -195,6 +198,14 @@ func GetConfig() *Config {
 		GatewayClusterServiceName:                    options.GetString(GATEWAY_CLUSTER_SERVICE_NAME),
 		PrometheusPushGateway:                        options.GetString(PROMETHEUS_PUSH_GATEWAY),
 	}
+
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
```

Comment:
Please consider hiding this logic behind a method:
https://github.com/RedHatInsights/app-common-go/issues/10